chapters/version_control.md

```diff
@@ -10,82 +10,77 @@ Recommended skill level: beginner - intermediate. Version control has a great de
 
 ## Table of contents
 
-1. [Summary](#Summary)
-2. [How version control is helpful](#How_version_control_is_helpful)
-3. [Version control: what it is and how it can be used to manage an evolving project](#what_version_control_is_and_how_it_can_be_used)
-    1. [What it is](#what_it_is)
-    2. [The basic workflow](#The_basic_workflow)
-    3. [Other facilities offered by version control](#branches_overview)
-4. [Why should you use version control?](#Why_should_you_use_version_control)
-5. [Getting Started](#Getting_started)
-6. [Commits](#Commits)
-    1. [The problem](#Commits_the_problem)
-    2. [The solution](#Commits_the_solution)
-    3. [How to do it](#adding)
-        1. [Retrieving past versions](#Retrieving_past_versions)
-    4. [Good practice](#Commits_good_practice)
-7. [Commit messages](#commit_messages)
-    1. [The problem](#Commit_messages_the_problem)
-    2. [The solution](#Commit_messages_the_solution)
-    3. [How to do it](#Commit_messages_how_to_do_it)
-    4. [Good practice](#Commit_messages_good_practice)
-8. [Comparing versions](#Comparing_versions)
-    1. [The problem](#Comparing_versions_the_problem)
-    2. [The solution](#Comparing_versions_the_solution)
-    3. [How to do it](#Comparing_versions_how_to_do_it)
-    4. [Good practice](#Comparing_versions_good_practice)
-9. [Branches](#Branches)
-    1. [The problem](#Branches_the_problem)
-    2. [The solution](#Branches_the_solution)
-    3. [How to do it](#Branches_how_to_do_it)
-    4. [Good practice](#Branches_good_practice)
+1. [Summary](#summary)
+2. [How version control is helpful](#how-version-control-is-helpful)
+3. [Version control: what it is and how it can be used to manage an evolving project](#version-control-what-it-is-and-how-it-can-be-used-to-manage-an-evolving-project)
+    1. [What it is](#what-it-is)
+    2. [The basic workflow](#the-basic-workflow)
+    3. [Other facilities offered by version control](#other-facilities-offered-by-version-control)
+4. [Why should you use version control?](#why-should-you-use-version-control)
+5. [Getting Started](#getting-started)
+6. [Commits](#commits)
+    1. [The problem](#the-problem)
+    2. [The solution](#the-solution)
+    3. [How to do it](#how-to-do-it)
+        1. [Retrieving past versions](#retrieving-past-versions)
+    4. [Good practice](#good-practice)
+7. [Commit messages](#commit-messages)
+    1. [The problem](#the-problem-2)
+    2. [The solution](#the-solution-2)
+    3. [How to do it](#how-to-do-it-2)
+    4. [Good practice](#good-practice-2)
```

Comment:
**Note:**  For duplicate heading names, GitHub resolves the name clash by appending `-2`, `-3`, etc., to the subsequent anchor names.  You may prefer the manually inserted anchors here.